src/Knp/DictionaryBundle/Command/DictionaryDumpCommand.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Knp\DictionaryBundle\Command;
+
+use Knp\DictionaryBundle\Dictionary;
+use Knp\DictionaryBundle\Dictionary\DictionaryRegistry;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Dump dictionaries with their related values
+ */
+class DictionaryDumpCommand extends ContainerAwareCommand
```

Comment:
You don't use the container so you can use the `Command` instead of the `ContainerAwareCommand`